### PR TITLE
manual test for PaymentResponse.payerPhone attr

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 06:59:39
Finished: 2017-08-21 07:04:25



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17593)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266694936)

File: payment-request/payment-response/payerPhone-attribute-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<script src="/resources/testharnessreport.js"></script>
+<script src="helpers.js"></script>
+<h2>payerPhone attribute</h2>
+<p>
+  When prompted, please use +12345678910 as the phone number.
+</p>
+<ol>
+  <li>
+    <button onclick="runManualTest(this, undefined, { payerPhone: null })">
+      payerPhone attribute is null when requestPayerPhone is undefined.
```
Comment (edenchuang) [inline payment-request/payment-response/payerPhone-attribute-manual.https.html]: Probably when PaymentOptions is undefined.

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-08-21 06:59:39
Finished: 2017-08-21 07:04:25



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17593)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/266694936)

**Review by edenchuang — APPROVED** [review]
LGTM

**Pull request merged**